.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -942,6 +950,58 @@ stages:
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: 2204-gen2-containerd
+  - stage: build_vhd_2404_containerd
+    dependsOn: []
```

Comment:
nit: can we just omit this?

I guess we do explicitly specify it this way everywhere else for some reason...

Reply:
we cant actually do this, removing the depends on takes an implicit dependency on previous stage for some reason
I guess the empty dependsOn basically says this stage doesn't depend on anything, run it. I will look at azure docs in more detail but looks like we need to keep it

<img width="697" alt="image" src="https://github.com/Azure/AgentBaker/assets/26747662/431e4518-005c-493b-8ca1-27b8fd5fe409">